src/EventListener/Dca/RendererDcaListener.php

```diff
@@ -125,20 +125,21 @@ public function getRenderSettings($dataContainer): array
 
         if ($dataContainer->activeRecord) {
             $repository = $this->repositoryManager->getRepository(LayerModel::class);
-            $layer      = $repository->find((int) $dataContainer->activeRecord->pid);
+            $layer      = $repository->find((int)$dataContainer->activeRecord->pid);
 
             if ($layer === null) {
                 return $settings;
             }
 
+            /** @var \Doctrine\DBAL\Statement $statement */
             $statement = $this->repositoryManager
                 ->getConnection()
                 ->prepare('SELECT id, name FROM tl_metamodel_rendersettings WHERE pid=:metaModelId');
 
             $statement->bindValue('metaModelId', $layer->metamodel);
-            $statement->execute();
+            $resutl = $statement->executeQuery();
```

Comment:
Please fix the typo:

```suggestion
            $result = $statement->executeQuery();
```